bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/messages.properties

```diff
@@ -26,4 +26,10 @@ InstallToolsHandler_InstallingToolsMsg=Installing tools...
 InstallToolsHandler_ItWilltakeTimeMsg=This can take a while. Please be patient.
 InstallToolsHandler_ToolsCompleted=Install tools completed.
 ListInstalledTools_MessageTitle=Message
-ListInstalledTools_MissingIdfPathMsg=Some tools are not installed or IDF_PATH is not set. To install them go to Help -> ESP-IDF Tools Manager -> Install Tools. Do you want to install the missing tools now?
\ No newline at end of file
+ListInstalledTools_MissingIdfPathMsg=Some tools are not installed or IDF_PATH is not set. To install them go to Help -> ESP-IDF Tools Manager -> Install Tools. Do you want to install the missing tools now?
+JavaRuntimeVersionMsg=Java runtime version: 
+OperationSystemMsg=Operation system: 
```

Comment:
typo - "Operating System"

Reply:
https://github.com/espressif/idf-eclipse-plugin/blob/cdaf7a1e2089498ec9e7f874cb037a9981f8ee60/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/messages.properties#L31
@kolipakakondal fixed that. Thx